index.php

```diff
@@ -453,11 +453,12 @@
                         <img src="images/icons/categories/64/preferences-system-parental-controls.svg" alt="Icon of an adult holding hand the hand of a child"/>
                         Parental Controls
                     </h2>
-                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Time Limits</h4>
+                    <h4>Implement parental controls or manage your own digital wellbeing.</h4>
+                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h3>Screen Time</h3>
                     <p>Set per-user time limits for weekdays, weekends, or both.</p>
-                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h4>Internet Use</h4>
+                    <?php include('images/icons/apps/symbolic/web-browser-symbolic.svg'); ?><h3>Internet Use</h3>
                     <p>Manage allowed websites. Rules affect all apps for the user, even if they use a different web browser.</p>
-                    <?php include('images/icons/actions/symbolic/view-grid-symbolic.svg'); ?><h4>Manage Apps</h4>
+                    <?php include('images/icons/actions/symbolic/view-grid-symbolic.svg'); ?><h3>Manage Apps</h3>
```

Comment:
This doesn't seem right. It's inconsistent with the Applications Menu section above. Also, changes to Parental Controls haven't been released. I think there's some issue with master, but you can't do anything with your own user on a clean install right now

Reply:
@danrabbit addressed :)